awx/ui_next/src/components/ListHeader/ListHeader.jsx

```diff
@@ -6,11 +6,10 @@ import { Toolbar, ToolbarContent } from '@patternfly/react-core';
 import DataListToolbar from '../DataListToolbar';
 
 import {
-  encodeNonDefaultQueryString,
   parseQueryString,
   mergeParams,
-  replaceParams,
   removeParams,
+  replaceNamespacedParams,
```

Comment:
I don't see a similar tweak to mergeParams here (what is used for certain searches, single select if I remember correct...I'm not sure if that is needed or not, I would test a single-select based search to be sure.

FWIW, I don't understand the point of the name change, just seems like more to type, it doesn't add any clarification in terms of the API, IMO.  I would keep the exported function as replaceParams (as it is clear), and change the original replaceParams to something else (and make it a non-exported helper function